android/app/build.gradle

```diff
@@ -51,33 +57,54 @@ android {
 
     signingConfigs {
         release {
-//      Commented to build without release keys
-//            keyAlias keystoreProperties['keyAlias']
-//            keyPassword keystoreProperties['keyPassword']
-//            storeFile file(keystoreProperties['storeFile'])
-//            storePassword keystoreProperties['storePassword']
+            keyAlias keystoreProperties['keyAlias']
+            keyPassword keystoreProperties['keyPassword']
+            storeFile file(keystoreProperties['storeFile'])
+            storePassword keystoreProperties['storePassword']
         }
     }
 
     buildTypes {
-        release {
-            // To release we need to switch to signingConfigs.release
+        debug {
+            // default
             signingConfig signingConfigs.debug
+        }
+
+        release {
+            // add --release to run command
+            signingConfig signingConfigs.release
 
             minifyEnabled true
             useProguard true
 
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
+
+    flavorDimensions "flavors"
+
+    productFlavors {
+        // Add particular setup into different app flavors.
+        // Also, you can create a different main to define global variables.
+        // Recommended read https://medium.com/flutter-community/flutter-ready-to-go-e59873f9d7de
+        development {
+            // To run  flutter run --flavor development -v lib/main.dart
+            dimension "flavors"
+            applicationId = "co.farmsmart.app.dev"
```

Comment:
Does this mean that debug and release apks can be installed in android at the same time without overriding each other?

Reply:
Exactly!